app/src/main/java/org/fossasia/openevent/general/auth/EditProfileFragment.kt

```diff
@@ -52,8 +52,14 @@ class EditProfileFragment : Fragment() {
             it?.let {
                 val userFirstName = it.firstName.nullToEmpty()
                 val userLastName = it.lastName.nullToEmpty()
+                val imageUrl = it.avatarUrl.nullToEmpty()
                 rootView.firstName.setText(userFirstName)
                 rootView.lastName.setText(userLastName)
+                Picasso.get()
+                    .load(imageUrl)
+                    .placeholder(AppCompatResources.getDrawable(context!!, R.drawable.ic_account_circle_grey_24dp)!!)
```

Comment:
Can you remove the exclamation marks and click on the red bulb that appears. 
You can then select the option to wrap it with ?. let call .

Reply:
Yes, I'm adding null checks to it right now. Thank you!

Reply:
I've created a new commit that adds a null check to the drawable and context. Is this okay?